docs/index.rst

```diff
@@ -489,19 +489,32 @@ You can also escape regular expressions by enclosing them within square brackets
 Punctuation
 ~~~~~~~~~~~
 
-Date formats may be fenced on either side by one punctuation character from the following list: :literal:`, . ; : ? ! " \` ' [ ] { } ( ) < >`
+Date and time formats may be fenced on either side by one punctuation character from the following list: ``, . ; : ? ! " \` ' [ ] { } ( ) < >``
 
 .. code-block:: python
 
-    >>> arrow.get("Tomorrow (2019-10-31) is Halloween!", "YYYY-MM-DD")
-    <Arrow [2019-10-31T00:00:00+00:00]>
+    >>> arrow.get("Tomorrow (2019-10-31T09:12:45) is Halloween!", "YYYY-MM-DDTHH:mm:ss")
+    <Arrow [2019-10-31T09:12:45+00:00]>
```

Comment:
Not bothered either way but why is this changed?

Reply:
I changed it because I noticed that the documentation only mentioned that dates can be fenced by punctuation, while both dates and times can. So I just tweaked one of the examples to include times :).